satpy/scene.py

```diff
@@ -1521,8 +1521,9 @@ def _generate_composite(self, comp_node: CompositorNode, keepables: set):
                 self._wishlist.remove(comp_node.name)
                 self._wishlist.add(cid)
             self._dependency_tree.update_node_name(comp_node, cid)
-        except IncompatibleAreas:
-            LOG.debug("Delaying generation of %s because of incompatible areas", str(compositor.id))
+        except (IncompatibleAreas, MissingTime):
+            LOG.debug("Delaying generation of %s because of incompatible areas "
+                      "or missing time dimension", str(compositor.id))
```

Comment:
Can we split this debug message or make it specific to the case (temporal versus area)?